robot_skills/src/robot_skills/torso.py

```diff
@@ -24,6 +21,7 @@ def __init__(self, robot_name, tf_listener):
         super(Torso, self).__init__(robot_name=robot_name, tf_listener=tf_listener)
 
         self.joint_names = self.load_param('skills/torso/joint_names')
+        self._arm_joint_names = self.load_param('skills/arm/joint_names')
```

Comment:
HACK TO THE MAX :cry:

Reply:
Extra comment has been added to indicated that this is unfortunately not going to work unless the entire concept of joint_groups is kicked out